doc/generate-request-schema.md

```diff
@@ -0,0 +1,51 @@
+# generate-request.json Schema
```

Comment:
Thank you for writing this! Would you mind moving it into the existing container-contract.md doc. I think it would be useful to see all that information together in one place.

Reply:
Done